managed/services/telemetry/config.default.yml

```diff
@@ -67,57 +59,189 @@ telemetry:
+    summary: "Agents version QAN_MYSQL_PERFSCHEMA_AGENT"
+    data:
+      - metric_name: "pmm_client_qan_mysql_perfschema_agent_version"
+        column: "version"
+
+  - id: NodesInfoQANMysqlSlowlogAgent
+    source: PMMDB_SELECT
+    query: version FROM agents WHERE 'agent_type = 'qan-mysql-slowlog-agent' and pmm_agent_id != 'pmm-server'
+    summary: "Agents version QAN_MYSQL_SLOWLOG_AGENT"
+    data:
+      - metric_name: "pmm_client_qan_mysql_slowlog_agent_version"
+        column: "version"
+
+  - id: NodesInfoQANMongodbProfilerAgent
     source: PMMDB_SELECT
-    query: agent_type, version FROM agents
-    summary: "Agents version"
+    query: version FROM agents WHERE 'agent_type = 'qan-mongodb-profiler-agent' and pmm_agent_id != 'pmm-server'
+    summary: "Agents version QAN_MONGODB_PROFILER_AGENT"
     data:
-      - metric_name: "nodes_info_agent"
-        column: "agent_type"
-      - metric_name: "nodes_info_version"
+      - metric_name: "pmm_client_qan_mongodb_profiler_agent_version"
         column: "version"
-      - metric_name: "classifier"
-        column: "agent_type"
+
+  - id: NodesInfoQANPostgresqlPgstatementsAgent
+    source: PMMDB_SELECT
+    query: version FROM agents WHERE 'agent_type = 'qan-postgresql-pgstatements-agent' and pmm_agent_id != 'pmm-server'
+    summary: "Agents version QAN_POSTGRESQL_PGSTATEMENTS_AGENT"
+    data:
+      - metric_name: "pmm_client_qan_postgresql_pgstatements_agent_version"
+        column: "version"
+
+  - id: NodesInfoQANPostgresqlPgstatmonitorAgent
+    source: PMMDB_SELECT
+    query: version FROM agents WHERE 'agent_type = 'qan-postgresql-pgstatmonitor-agent' and pmm_agent_id != 'pmm-server'
+    summary: "Agents version QAN_POSTGRESQL_PGSTATMONITOR_AGENT"
+    data:
+      - metric_name: "pmm_client_qan_postgresql_pgstatmonitor_agent_version"
+        column: "version"
+
+  - id: NodesInfoRDSExporter
+    source: PMMDB_SELECT
+    query: version FROM agents WHERE 'agent_type = 'rds-exporter' and pmm_agent_id != 'pmm-server'
+    summary: "Agents version RDS_EXPORTER"
+    data:
+      - metric_name: "pmm_client_rds_exporter_version"
+        column: "version"
+
+  - id: NodesInfoExternalExporter
+    source: PMMDB_SELECT
+    query: version FROM agents WHERE 'agent_type = 'external-exporter' and pmm_agent_id != 'pmm-server'
+    summary: "Agents version EXTERNAL_EXPORTER"
+    data:
+      - metric_name: "pmm_client_external_exporter_version"
+        column: "version"
+
+  - id: NodesInfoAzureDatabaseExporter
+    source: PMMDB_SELECT
+    query: version FROM agents WHERE 'agent_type = 'azure-database-exporter' and pmm_agent_id != 'pmm-server'
+    summary: "Agents version AZURE_DATABASE_EXPORTER"
+    data:
+      - metric_name: "pmm_client_azure_database_exporter_version"
+        column: "version"
+
+  - id: PMMNodesInfoNodeType
+    source: PMMDB_SELECT
+    query: node_type FROM nodes WHERE node_name = 'pmm-server'
```

Comment:
@rnovikovP what is the sense in this metric? this value is always 'generic', isn't it?

Reply:
For now .. maybe  but we adding more  deployment ways so this might be a was to differentiate them later

Reply:
but we have a separate metric 'deployment_method'

Reply:
as an additional  verification method - will be ok , IMO